### ISSUE TITLE
Tram regressed AGAIN and has two circuit labs / cargo consoles AGAIN

### ISSUE THREAD

**Issue by MrMelbert** [tgstation/tgstation#70698]
## Reproduction:

![image](https://user-images.githubusercontent.com/51863163/197086194-ca60c055-bf59-4877-b573-59aa3ea94617.png)

![image](https://user-images.githubusercontent.com/51863163/197086216-2a62c688-4938-4102-b239-a32c3c043dbb.png)

![image](https://user-images.githubusercontent.com/51863163/197086393-e441113b-ea38-4169-ae44-394207afff0c.png)

![image](https://user-images.githubusercontent.com/51863163/197086402-1ea59faa-1600-426f-85dc-bc839c635354.png)

#68073 
#68459

**Comment by msgerbs** [comment]
_How?_ Is somebody editing tram without updating their repo and not using mapmerge?

**Comment by Sealed101** [comment]
its the camera goblin that haunts mmmiracles im sure of it

**Comment by ShizCalev** [comment]
i'd assume #69000

**Comment by moocowswag** [comment]
This is the second time tramstation had a reoccuring issue that was circuits related.

**Comment by MMMiracles** [comment]
this is no longer relevant after #72331
